From 3b83bc7175c14c56be01bdbd809c3f2a341d4fd5 Mon Sep 17 00:00:00 2001
Message-Id: <3b83bc7175c14c56be01bdbd809c3f2a341d4fd5.1368111913.git.minovotn@redhat.com>
In-Reply-To: <405603258af5154387bea676be1f904b6713f6ae.1368111913.git.minovotn@redhat.com>
References: <405603258af5154387bea676be1f904b6713f6ae.1368111913.git.minovotn@redhat.com>
From: Amit Shah <amit.shah@redhat.com>
Date: Wed, 24 Apr 2013 08:17:58 +0200
Subject: [PATCH 24/65] char: rename qemu_chr_close() -> qemu_chr_delete()

RH-Author: Amit Shah <amit.shah@redhat.com>
Message-id: <b722c885ae09d58bb208aaef82f67cb4a3de3282.1366724981.git.amit.shah@redhat.com>
Patchwork-id: 50802
O-Subject: [RHEL6.5 qemu-kvm PATCH 24/65] char: rename qemu_chr_close() -> qemu_chr_delete()
Bugzilla: 909059
RH-Acked-by: Hans de Goede <hdegoede@redhat.com>
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

From: Anthony Liguori <aliguori@us.ibm.com>

Signed-off-by: Anthony Liguori <aliguori@us.ibm.com>
(cherry picked from commit 70f24fb6c647c65760c8e1caa29f72531d5be8cd)

Signed-off-by: Amit Shah <amit.shah@redhat.com>

Conflicts:
	gdbstub.c
	qemu-char.h
	usb-redir.c

Signed-off-by: Amit Shah <amit.shah@redhat.com>
---
 gdbstub.c               | 2 +-
 hw/ccid-card-passthru.c | 2 +-
 hw/usb-serial.c         | 2 +-
 qemu-char.c             | 4 ++--
 qemu-char.h             | 2 +-
 usb-redir.c             | 2 +-
 6 files changed, 7 insertions(+), 7 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 gdbstub.c               | 2 +-
 hw/ccid-card-passthru.c | 2 +-
 hw/usb-serial.c         | 2 +-
 qemu-char.c             | 4 ++--
 qemu-char.h             | 2 +-
 usb-redir.c             | 2 +-
 6 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/gdbstub.c b/gdbstub.c
index 3a42ccf..d447115 100644
--- a/gdbstub.c
+++ b/gdbstub.c
@@ -2590,7 +2590,7 @@ int gdbserver_start(const char *device)
         monitor_init(mon_chr, 0);
     } else {
         if (s->chr)
-            qemu_chr_close(s->chr);
+            qemu_chr_delete(s->chr);
         mon_chr = s->mon_chr;
         memset(s, 0, sizeof(GDBState));
     }
diff --git a/hw/ccid-card-passthru.c b/hw/ccid-card-passthru.c
index 77d2027..9ba6969 100644
--- a/hw/ccid-card-passthru.c
+++ b/hw/ccid-card-passthru.c
@@ -199,7 +199,7 @@ static void ccid_card_vscard_handle_message(PassthruState *card,
 
 static void ccid_card_vscard_drop_connection(PassthruState *card)
 {
-    qemu_chr_close(card->cs);
+    qemu_chr_delete(card->cs);
     card->vscard_in_pos = card->vscard_in_hdr = 0;
 }
 
diff --git a/hw/usb-serial.c b/hw/usb-serial.c
index ef87025..d4ed4e3 100644
--- a/hw/usb-serial.c
+++ b/hw/usb-serial.c
@@ -422,7 +422,7 @@ static void usb_serial_handle_destroy(USBDevice *dev)
 {
     USBSerialState *s = (USBSerialState *)dev;
 
-    qemu_chr_close(s->cs);
+    qemu_chr_delete(s->cs);
 }
 
 static int usb_serial_can_read(void *opaque)
diff --git a/qemu-char.c b/qemu-char.c
index 476be31..4764a2d 100644
--- a/qemu-char.c
+++ b/qemu-char.c
@@ -2322,7 +2322,7 @@ QString *qemu_chr_mem_to_qs(CharDriverState *chr)
     return qstring_from_substr((char *) d->outbuf, 0, d->outbuf_size - 1);
 }
 
-/* NOTE: this driver can not be closed with qemu_chr_close()! */
+/* NOTE: this driver can not be closed with qemu_chr_delete()! */
 void qemu_chr_close_mem(CharDriverState *chr)
 {
     MemoryDriver *d = chr->opaque;
@@ -2596,7 +2596,7 @@ void qemu_chr_fe_close(struct CharDriverState *chr)
     }
 }
 
-void qemu_chr_close(CharDriverState *chr)
+void qemu_chr_delete(CharDriverState *chr)
 {
     QTAILQ_REMOVE(&chardevs, chr, next);
     if (chr->chr_close)
diff --git a/qemu-char.h b/qemu-char.h
index 7ca5e14..401de83 100644
--- a/qemu-char.h
+++ b/qemu-char.h
@@ -83,7 +83,7 @@ CharDriverState *qemu_chr_new(const char *label, const char *filename, void (*in
 void qemu_chr_fe_set_echo(struct CharDriverState *chr, bool echo);
 void qemu_chr_fe_open(struct CharDriverState *chr);
 void qemu_chr_fe_close(struct CharDriverState *chr);
-void qemu_chr_close(CharDriverState *chr);
+void qemu_chr_delete(CharDriverState *chr);
 void qemu_chr_fe_printf(CharDriverState *s, const char *fmt, ...);
 int qemu_chr_fe_write(CharDriverState *s, const uint8_t *buf, int len);
 void qemu_chr_send_event(CharDriverState *s, int event);
diff --git a/usb-redir.c b/usb-redir.c
index 1388b0b..f8d8dab 100644
--- a/usb-redir.c
+++ b/usb-redir.c
@@ -1096,7 +1096,7 @@ static void usbredir_handle_destroy(USBDevice *udev)
     USBRedirDevice *dev = DO_UPCAST(USBRedirDevice, dev, udev);
 
     qemu_chr_fe_close(dev->cs);
-    qemu_chr_close(dev->cs);
+    qemu_chr_delete(dev->cs);
     /* Note must be done after qemu_chr_close, as that causes a close event */
     qemu_bh_delete(dev->chardev_close_bh);
 
-- 
1.7.11.7

